blog/en/blog/2022/10/27/ten-use-cases-api-gateway.md

```diff
@@ -0,0 +1,180 @@
+---
```

Comment:
Nitpicking: diagrams all have different styles. Can you choose a style and keep it consistent across all diagrams?

Reply:
Yes, styles are going to be changed.

Reply:
@nfrankel and @SylviaBABY now I updated all pictures that had different styles and we are good to go.